### PR TITLE
Override browser default tag based styles that use EM units

### PR DESCRIPTION
**Pull request by kof**
## Description

EM seems to be challenging to use and people tend to use PX or REM instead ... I changed those values that come with EM by default from browsers to the values Webflow is using.

I am not sure why webflow is doing certain things: e.g. paragraphs have marginTop 0, h1-3 have marginTop 20; marginBottom: 10, h4- have marginTop 10 ; marginBottom 20

These seem to be inconsistent and weird choices.

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio-builder/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env.example` and the `builder/env-check.js` if mandatory

### CODE REVIEW

**Review by istarkov — COMMENTED** [review]
It doesn't look like normalize now. Its set of opinionated styles has nothing with initial behavior of normalize - makes browsers render all elements more consistently

**Comment by istarkov** [comment]
I think if we do changes which are different from notmalize.css we need to do them in other place

**Comment by kof** [comment]
> It doesn't look like normalize now. Its set of opinionated styles has nothing with initial behavior of normalize - makes browsers render all elements more consistently

yes which is why the opinionated styles are in a separate file, but we use them from normalize ... should probably  merge them together in a different way instead of using reset directly from normalize, but that's not new here .. this was done before ... may need to change this @TrySound

**Comment by taylornowotny** [comment]
One issue - the font and height properties have an orange label with no Value source in the tooltip. Orange labels represent remote user-added values so they should always show the value source in the tooltip. But these orange labels show on a new project with no user styles. These properties should be gray labels (Webstudio default) instead.

Screenshot 2023-07-03 at 9 07 06 AM
Screenshot 2023-07-03 at 9 09 44 AM

**Comment by kof** [comment]
After a discussion we decided to not have margins by default at all

File: packages/react-sdk/src/css/presets.ts [hunk 1]
```diff
@@ -25,3 +25,102 @@ export const outline: Styles = [
...
+    value: { type: "keyword", value: "solid" },
+  },
+  {
+    property: "borderLeftColor",
+    value: { type: "rgb", r: 226, g: 226, b: 226, alpha: 1 },
+  },
+] satisfies Styles;
+
+export const h1 = [
+  ...margins,
```
Comment (TrySound) [inline packages/react-sdk/src/css/presets.ts:81]: headers have only top and bottom margins set. Do we want to bloat styles with left right too?

File: packages/react-sdk/src/css/presets.ts [hunk 2]
```diff
@@ -25,3 +25,102 @@ export const outline: Styles = [
...
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+  {
+    property: "marginRight",
+    value: { type: "unit", value: 0, unit: "number" },
+  },
+  {
+    property: "marginBottom",
+    value: { type: "unit", value: 0, unit: "px" },
+  },
```
Comment (TrySound) [inline packages/react-sdk/src/css/presets.ts:41]: why number and px?

**Comment by TrySound** [comment]
> this was done before ... may need to change this @TrySound

let's do it later

File: packages/react-sdk/src/css/presets.ts (hunk 2 above)
Comment (kof) [reply packages/react-sdk/src/css/presets.ts:41]: typo, please change to number, I am off right now

**Review by TrySound — APPROVED** [review]

File: packages/react-sdk/src/css/normalize.ts [hunk 3]
```diff
@@ -37,13 +37,21 @@ const boxSizing = {
...
 export const div = baseStyle;
 export const address = baseStyle;
 export const article = baseStyle;
 export const aside = baseStyle;
-export const figure = baseStyle;
+export const blockquote = [
+  ...baseStyle,
+  ...presets.blockquote,
+] satisfies Styles;
+export const figure = [...baseStyle, ...presets.margins] satisfies Styles;
```
Comment (kof) [inline packages/react-sdk/src/css/normalize.ts:54]: might not need margins, but only vertical margins, need to check browser defaults there

**Pull request merged**